src/pal/src/map/virtual.cpp

```diff
@@ -137,7 +137,9 @@ namespace VirtualMemoryLogging
         IN LPVOID returnedAddress,
         IN BOOL result)
     {
-        LONG i = InterlockedIncrement(&recordNumber) - 1;
+        ULONG i = InterlockedIncrement((LONG *)&recordNumber) - 1;
```

Comment:
Can you please put cast to ULONG before the InterlockedIncrement too? Clang is probably ok with it, but some people spent quite an effort trying to keep the codebase buildable with GCC too and it is picky on these.

Reply:
will fix this